explorer/src/queries/search.ts

```diff
@@ -54,14 +62,14 @@ const searchBuilder = (params: SearchParams): SelectQueryBuilder<JobRun> => {
 }
 
 export const search = async (params: SearchParams): Promise<JobRun[]> => {
-  return searchBuilder(params)
+  return pagedSearchBuilder(params)
     .leftJoinAndSelect('job_run.chainlinkNode', 'chainlink_node')
     .orderBy('job_run.createdAt', 'DESC')
     .getMany()
 }
 
 export const count = async (params: SearchParams): Promise<number> => {
```

Comment:
RE: https://github.com/smartcontractkit/chainlink/pull/2750#issuecomment-618686343

Gotcha. How do you feel about restricting the type at the boundary? 

```typescript
export const search = async (params: Pick<SearchParams, 'searchQuery'>): Promise<JobRun[]> => {
```

Reply:
Yes, this is 😎